### ISSUE TITLE
question about paper：disjoint and overlapped

### ISSUE THREAD

**Issue by WAMAWAMA** [fcdl94/MiB#25]
Hi，this is a really interesting work，but when reading your manuscript，i am confused about the two setups: *overlapped* and *disjoint*, can you describe it with much more details?

**Comment by fcdl94** [comment]
Hi @WAMAWAMA,
Sorry for the delay in answering you.
Basically, previous work operated in two different ways, both realistic and interesting.
The first, the disjoint setup, was proposed by Michieli et at. In this setting, the images containing classes seen in the future (e.g. sheep in 15-5 setting) are discarded, and the training is performed only on pixels of seen classes.
The second, the overlapped setup, was proposed by Shemelkov et al. in the context of object detection. In this setting, all the images containing known classes are considered. For example, in 15-5, if an image contains pixels of sheep, which is a future class, but also pixels of person that is a known class, the image is considered.
So, to summarize, the main change relies on the images considered in the training steps. The disjoint setup never sees pixels of future classes, while in the overlapped there may be present pixels as future classes (but labeled as background, since they are not known yet).

**Comment by fcdl94** [comment]
Hi @WAMAWAMA. Have you understood the difference? Can I close the issue?

**Comment by WAMAWAMA** [comment]
@fcdl94 thanks for your clear and understandable explanation!👍, i've already understood, and sorry for replying so late

**Comment by jumptiger66** [comment]
so wonderful explanation.
